[2024-05-31T06:09:34.392+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-31T06:09:34.415+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_weather_workflow_DAG_v04.group_APIs.load_csv_to_postgres manual__2024-05-31T05:49:47.133585+00:00 [queued]>
[2024-05-31T06:09:34.424+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_weather_workflow_DAG_v04.group_APIs.load_csv_to_postgres manual__2024-05-31T05:49:47.133585+00:00 [queued]>
[2024-05-31T06:09:34.425+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-05-31T06:09:34.439+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): group_APIs.load_csv_to_postgres> on 2024-05-31 05:49:47.133585+00:00
[2024-05-31T06:09:34.447+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=4624) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-31T06:09:34.448+0000] {standard_task_runner.py:63} INFO - Started process 4626 to run task
[2024-05-31T06:09:34.447+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'my_weather_workflow_DAG_v04', 'group_APIs.load_csv_to_postgres', 'manual__2024-05-31T05:49:47.133585+00:00', '--job-id', '258', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmp5_no7gn4']
[2024-05-31T06:09:34.450+0000] {standard_task_runner.py:91} INFO - Job 258: Subtask group_APIs.load_csv_to_postgres
[2024-05-31T06:09:34.518+0000] {task_command.py:426} INFO - Running <TaskInstance: my_weather_workflow_DAG_v04.group_APIs.load_csv_to_postgres manual__2024-05-31T05:49:47.133585+00:00 [running]> on host cd9caf1c47f0
[2024-05-31T06:09:34.640+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Khang' AIRFLOW_CTX_DAG_ID='my_weather_workflow_DAG_v04' AIRFLOW_CTX_TASK_ID='group_APIs.load_csv_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2024-05-31T05:49:47.133585+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-31T05:49:47.133585+00:00'
[2024-05-31T06:09:34.642+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-31T06:09:34.700+0000] {base.py:84} INFO - Using connection ID 'postgres_connection_id' for task execution.
[2024-05-31T06:09:34.716+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-31T06:09:34.717+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-31T06:09:34.738+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=my_weather_workflow_DAG_v04, task_id=group_APIs.load_csv_to_postgres, run_id=manual__2024-05-31T05:49:47.133585+00:00, execution_date=20240531T054947, start_date=20240531T060934, end_date=20240531T060934
[2024-05-31T06:09:34.784+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-31T06:09:34.842+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-31T06:09:34.845+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
